versionner/version.py

```diff
@@ -224,7 +224,7 @@ def write(self, version):
             shutil.copystat(str(self._path), fh.name)
 
         try:
-            pathlib.Path(fh.name).rename(self._path)
+            shutil.move(fh.name, self._path)
         except OSError as exc:
```

Comment:
Could you please remove also this `try/except` block, because it's not necessary here anymore?